### PR TITLE
Add Invert Image Tool

### PR DESCRIPTION
**Pull request by wgranger**
The invert tool allows users to invert the colors of an image/subject in order to more easily notice objects. This branch was tested and should be working on all browsers (IE, Safari, Firefox, Chrome). I tried to abstract the invert into a modification property to allow the `ModifiedImage` component to be more flexible with other filters in the future.

Staged branch at https://invert-tool.pfe-preview.zooniverse.org/projects/wgranger-test/invert-testing/classify

### CODE REVIEW

File: app/components/modified-image.cjsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        </feComponentTransfer>
+      </filter>
+    </defs>
+  </svg>'
+
+module.exports = React.createClass
+  displayName: 'ModifiedImage'
+
+  componentWillMount: ->
+    @setElements()
```
Comment (brian-c) [inline app/components/modified-image.cjsx]: You should do this once, outside the component lifecycle. As written, it'll add a new `#svg-invert-filter` every time one of these mounts.

**Comment by wgranger** [comment]
@brian-c Outside the component _lifecycle_ makes sense, but also keep that HTML code in the `ModifiedImage` component? Seems like it would be more concise that way.

**Comment by wgranger** [comment]
Staged branch has been updated.

**Comment by brian-c** [comment]
Looks good to me! Thanks.

**Pull request merged**